event-bus/cassandra/src/main/java/org/apache/james/events/CassandraEventDeadLetters.java

```diff
@@ -19,23 +19,44 @@
 
 package org.apache.james.events;
 
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
 import jakarta.inject.Inject;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableSet;
 
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 public class CassandraEventDeadLetters implements EventDeadLetters {
```

Comment:
Are we really going to duplicate the Event -> Json and json -> event in all classes like PG or we do plan to have an abstraction we can reuse?